modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1131,8 +1021,10 @@ AUI.add(
 
 						var datePicker = instance.getDatePicker();
 
+						var value;
+
 						if (!datePicker) {
-							return '';
+							value = '';
 						}
 
 						var selectedDate = datePicker.getDate();
```

Comment:
Hey @rodrigopaulino , After the change at line 1135, we will get an exception at line 1138 due to `datePicker` be undefined.  Please, fix it.

Thx